server/src/core/server/graph/resolvers/UserStatus.ts

```diff
@@ -57,6 +58,10 @@ export const UserStatus: Required<GQLUserStatusTypeResolver<UserStatusInput>> =
       ...user.consolidateUsernameStatus(username),
       userID,
     }),
+    deletion: ({ userID, deletion }): UserDeletionStatusInput => ({
```

Comment:
Same here, seems like the resolver might have generated a result type instead of an Input.